src/datatransfer.js

```diff
@@ -42,4 +50,26 @@ export default class DataTransfer {
 	setData( type, data ) {
 		this._native.setData( type, data );
 	}
+
+	*_getFiles() {
```

Comment:
Why is this function a generator if above we do `Array.from()` anyway? It doesn't make sense.